### PR TITLE
PAR-FTWD-alex

### PR DESCRIPTION
**Pull request by alexandfox**
apple recipe changes

### CODE REVIEW

File: starter-code/apple-pie-starter-code/css/main.css [hunk 1]
```diff
@@ -0,0 +1,80 @@
+* {
+	margin: 0;
+	padding: 0;
+	box-sizing: border-box;
+}
+
+.hero {
```
Comment (ta-web-paris) [inline starter-code/apple-pie-starter-code/css/main.css]: it's better, when possible to give a meaningful names to your classes. This is going to be really important for when you have a lot of css

File: starter-code/apple-pie-starter-code/css/main.css [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+	list-style-type: none;
+	counter-reset: item;
+}
+
+.centered #ingredients .ingredient:before {
+	content: "- ";
+}
+
+
+.centered #directions .direction:before {
```
Comment (ta-web-paris) [inline starter-code/apple-pie-starter-code/css/main.css]: This is super cool!

**Comment by ta-web-paris** [comment]
your recipe clone was really good! 🙌

File: starter-code/apple-pie-starter-code/css/main.css (hunk 1 above)
Comment (alexandfox) [reply starter-code/apple-pie-starter-code/css/main.css]: hero is the banner image on the webpage! usually at the top of the page/ full width

File: starter-code/apple-pie-starter-code/css/main.css (hunk 2 above)
Comment (alexandfox) [reply starter-code/apple-pie-starter-code/css/main.css]: is there another way to change the bullet point types without doing this? what is the expected format for styling them?

**Pull request closed without merging**